scripts/clang-tidy-diff.sh

```diff
@@ -27,11 +14,11 @@ if [ ! -e "$CLANG_TIDY_DIFF" ]; then
   echo "Failed to find clang-tidy-diff.py ($CLANG_TIDY_DIFF)"
   exit 1
 fi
-TIDY_MSG=$(git diff -U0 $BRANCH... | $CLANG_TIDY_DIFF -quiet -p1 2> /dev/null)
+TIDY_MSG=$(git diff -U0 origin/$GITHUB_BASE_REF | $CLANG_TIDY_DIFF -quiet -p1 2> /dev/null)
```

Comment:
Can we not keep $BRANCH (perhaps call it $BASE if you like) and have it default to something that works when run locally?

Reply:
Ah, good idea 👍